lib/Selenium/ActionChains.pm

```diff
@@ -336,7 +336,7 @@ LIMITATIONS.
 =head2 key_down
 
 Sends key presses only, without releasing them.
-Useful when modifier keys are requried
+Useful when modifier keys are requiried
```

Comment:
This is still incorrect (extra i).  The actual spelling is "required".  It was a simple transposition.